2016-04-28 21:20:25,587 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/opt/py27environment/lib/python2.7/site-packages/streamparse/storm/component.py", line 487, in run
    self.initialize(storm_conf, context)
  File "bolts/wordcount.py", line 12, in initialize
    conn = psycopg2.connect(database="tcount", user="postgres", password="pass", host="localhost", port="5432")
  File "/opt/py27environment/lib/python2.7/site-packages/psycopg2/__init__.py", line 164, in connect
    conn = _connect(dsn, connection_factory=connection_factory, async=async)
OperationalError: FATAL:  database "tcount" does not exist

